### ISSUE TITLE
Fix accessibility in IPP flows

### ISSUE THREAD

**Issue by malinajirka** [woocommerce/woocommerce-android#13250]
Goal of this task is to ensure IPP Flows (payments, welcome, onboarding) work as expected on devices that increase font size using accessibility setting in the OS. 

Suggested solution is to wrap the content in a scrollview to at least ensure the widgets don't overlap. 

~In case, this solution would turn out to have side-effects, we can also consider using `dp` instead of `sp` for the texts to at least ensure changing the accessibility setting won't break the layout.~ Update: We decided this workaround is not acceptable as it's not accessible.

<img width="384" alt="Image" src="https://github.com/user-attachments/assets/9614fc8b-bc24-4910-9fc9-eef200904c86" />

**Comment by dangermattic** [comment]
Thanks for reporting! 👍

**Comment by Akshaykomar890** [comment]
@malinajirka Can I work on this issue ?

**Comment by malinajirka** [comment]
@Akshaykomar890 Thanks again for your interest in contributing!

Feel free to work on this issue, but beware that it likely won't be a simple or a quick one. These flows require quite a bit of setup on both the client as well as the test site. Moreover, there are many different flows (the ones mentioned above are not all the flows that require fixing). These flows are quite prone-to-breaking since the OS runs default animation that can't be easily disabled if dimensions of the dialogs differ even by one pixel, so we need to keep all the dialogs the exact same dimensions.

I don't want to discourage you too much, but at the same time I feel it's only fair to warn you about the complexity of this issue. Feel free to work on this one as well as feel free to pick a different one.

**Comment by Akshaykomar890** [comment]
Thank you @malinajirka  for the detailed explanation and for taking the time to outline the challenges of this issue. After considering the complexity and my current experience level as a beginner, I believe it would be best for me to start with smaller issues to gain a better understanding of the project. I’ll focus on those first and revisit more complex tasks like this one when I’ve built up more familiarity. Thank you again for your guidance
